File: trabalho 1/clima.java
class Clima {
    public String ano;
    public String mes; 
    public String temperatura;
    public String precipitacao;

    public Clima(String ano, String mes, String temperatura, String precipitacao) {
        this.ano = ano;
        this.mes = mes;
        this.temperatura = temperatura;
        this.precipitacao = precipitacao;
    }

    @Override
    public String toString() {
        return "Clima [ano=" + this.ano + ", mes=" + this.mes + 
               ", temperatura=" + this.temperatura + "°C, precipitacao=" + this.precipitacao + "]";
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) return true;
        if (obj == null || getClass() != obj.getClass()) return false;
        Clima c = (Clima) obj;
        return this.ano.equals(c.ano) && this.mes.equals(c.mes);
    }
}

File: aula 5/Python/Clima.py
class Clima: 
    def __init__(self, ano, mes, temperatura, precipitacao):
        self.ano = ano
        self.mes = mes
        self.temperatura = temperatura
        self.precipitacao = precipitacao

    def __str__(self):
        return f"Dados climaticos:Ano:{self.ano}. Mes:{self.mes}. Temperatura:{self.temperatura}. Precipitacao:{self.precipitacao}"

    def __eq__(self, object):
        return self.ano == object.ano and self.mes == object.mes
